ui/src/utils/i18n.js

```diff
@@ -4,36 +4,28 @@ import LanguageDetector from 'i18next-browser-languagedetector';
 import _ from 'lodash';
 
 const resources = {};
-const languages = ['en', 'zh_CN'];
 
-function loadResource(res) {
-  languages.forEach(lang => {
-    _.merge(resources, { [lang]: { translation: res[lang] } });
+export function loadResourceFromRequireContext(requireContext) {
```

Comment:
I removed this function in https://github.com/pingcap-incubator/tidb-dashboard/commit/97c96ad10cecf342b33dc4d70a2f6a918e0a8ca5, replace d it by i18n.addResourceBundle() after init() to support i18n for left nav menus.

Reply:
Would you mind modifying this PR directly for the changes you propose? Since looks like there will be some conflicts.

Reply:
Yep, let me update it tmw.

Reply:
I have helped you merge the branch, PTAL 

https://github.com/pingcap-incubator/tidb-dashboard/pull/70/commits/2472f2828f5005dd7617752564c8c53fa405d32f